src/index.css

```diff
@@ -1,27 +1,306 @@
+  align-items: center;
+  background-color: white;
+  display: -webkit-box;
+  display: flex;
+  font-family: 'Montserrat Bold', sans-serif;
+  height: 5.938rem;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  margin: 0 auto;
+}
+
+header .header-text {
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  display: -webkit-box;
+  margin-left: 3rem;
+}
+
+header .header-text > * {
+  margin-right: 1.5rem;
+}
+
+header .bookstore-cms {
+  margin-left: 4rem;
+  color: #0290ff;
+  font-size: 1.875rem;
+  font-weight: bold;
+}
+
+header .books {
+  color: #121212;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+}
+
+header .categories {
+  color: #121212;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  opacity: 0.5;
+}
+
+header .icon {
+  margin-right: 8rem;
+  color: #0290ff;
+  background-color: white;
+  border-radius: 50%;
+  border: 1px solid #e5e7eb;
+  padding: 10px;
+}
+
+.select-dropdown {
+  margin: 1rem auto;
+  width: 100%;
+}
+
+.select-dropdown-2,
+.form-submit {
+  width: 18%;
+}
+
+.book-input {
+  width: 60%;
+  background-color: #fff;
+  border-color: #e5e7eb;
+  border-radius: 4px;
+  border-style: solid;
+  border-width: 1px;
+  line-height: 24px;
+  padding: 12px 16px;
+}
+
+.lesson-panel {
+  width: 70rem;
+  height: 12rem;
+  margin: 1rem  0 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  border: solid 1px rgb(240, 240, 240);
+}
+
+.book-info {
+  grid-area: auto / span 3 / auto / span 3;
+  line-height: 24px;
+  text-align: left;
+}
+
+.category {
+  color: #121212;
+  font-family: 'Montserrat Bold', sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 20px;
+  text-align: left;
+  text-transform: capitalize;
+}
+
+.title {
+  font-family: 'Roboto Slab', serif;
+  font-size: 22px;
+  font-weight: 700;
+  letter-spacing: -0.2px;
+  margin: 0 0 4px;
+}
+
+.id {
+  color: #4386bf;
+  font-size: 14px;
+  font-weight: 300;
+  line-height: 20px;
+  margin: 0  0 16px;
+  text-align: left;
+}
+
+.button-grp {
+  margin: 32px 0 0;
+}
+
+.remove {
+  padding: 0 16px;
+  color: #4386bf;
+}
+
+.comments .edit .remove {
+  align-items: flex-start;
+  display: inline-block;
+  font-family: 'Roboto Slab', serif;
+  font-size: 14px;
+  font-weight: 300;
+  line-height: 20px;
+  text-align: center;
+}
+
+.comments {
+  padding: 0 16px 0 0;
+  color: #4386bf;
+  border-right: 1px solid #e5e7eb;
+}
+
+.edit {
+  padding: 0 0 0 16px;
+  color: #4386bf;
+  border-left: 1px solid #e5e7eb;
+}
+
+.progress-info {
+  align-items: center;
+  display: flex;
+  grid-area: auto / span 2 / auto / span 2;
+  justify-content: center;
+  line-height: 24px;
+  text-align: left;
+  border-right: 1px solid #e5e7eb;
+}
+
+/* .book-progress {
+  display: flex;
+  border-right: 0.1px solid rgb(240, 240, 240);
+  padding-right: 1rem;
+} */
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.